09_variables_conditionals/solution_student/temp_converter/js/main.js

```diff
@@ -0,0 +1,48 @@
+
+
+function convertFtoC(fTemp) {
+	var fTemp = (fTemp - 32) * (5/9);
+	return fTemp;
+};
+
+$('#fahrenheit_to_celsius').on('click', function() {
+	var typedTemp = $('#temperature').val();
+	typedTemp = parseFloat(typedTemp);
+	var resultTemp = convertFtoC(typedTemp);
+console.log(resultTemp);
+$('#result').html(resultTemp);
+
+});
+
+//I thought I was going in the right direction here to but every 
```

Comment:
looks like the code from my solution file (my fault, haha), but I'd have to see how you were trying to implement it to know why it was breaking